pkg/bdba/ctl_bdba.go

```diff
@@ -136,6 +136,60 @@ type FlagTree struct {
 	PGExistingSecret string `json:"pgExistingSecret"`
 }
 
+// DefaultFlagTree ...
+// [Dev Note]: These should match the Helm Chart's Values.yaml
+var DefaultFlagTree = FlagTree{
+	// Version
+	Version: "2020.03",
+	// Cluster Domain
+	ClusterDomain: "cluster.local",
+	// Storage
+	PGStorageClass:       "default",
+	PGSize:               "300Gi",
+	MinioStorageClass:    "default",
+	MinioSize:            "300Gi",
```

Comment:
Can you pls change all of them to Claim size?

Reply:
Done!